ProjectTaskTracker.js

```diff
@@ -60,15 +65,12 @@ if(tasks){
 // 	}
 // }	
 
+//I NEED HELP HERE!
```

Comment:
With what? Comment what needs to happen here, add `// TODO`. Try to get as much info as you can first